.github/workflows/fuzzer.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - name: Initializing fuzzer with previous inputs 
       run: |
         cd fuzzer
-        HFUZZ_RUN_ARGS="--dict=json.dict --run_time 10800 --timeout 60 -T" cargo hfuzz run fuzz_json
+        HFUZZ_RUN_ARGS="--dict=json.dict --run_time 10800 --timeout 60" cargo hfuzz run fuzz_json
```

Comment:
Removed the `-T` flag so that timeouts aren't flagged as errors. This change is needed because
1. Sometimes running honggfuzz is just slower than the release or debug version of the program
2. There is currently no way to tell if the program hangs because of a correct infinite loop